docker-compose.yml

```diff
@@ -1,11 +1,10 @@
 name: e2e-tests
 services:
   playwright-e2e:
-    image: ecmchow/node-playwright-pnpm:v1.45.3-node20-pnpm9-jammy
-    command: sh -c "pnpm install && pnpm e2e $${UPDATE_SNAPSHOTS}"
+    image: ecmchow/node-playwright-pnpm:v1.46.1-node20-pnpm9-jammy
```

Comment:
Previously I got a log that indicated that 46.1 was not the version that our playwright version was expecting, but I guess we just updated that dependency in your other PR right? so nice catch!

Reply:
We should set some configuration to force rebase before merging into `main` branch to pick up the latest changes

Reply:
Hmm unfortunately that does not seem to exist. There is the option to require an up to date branch (and that is enabled) but you could update through merge or rebase. I would want that to be rebase only, but that does not seem possible.